tests/confreq-rs-xml-nval_comment/OPS/package.opf

```diff
@@ -4,14 +4,14 @@
     <dc:coverage>Publication Resources</dc:coverage>
     <dc:creator>Dave Cramer</dc:creator>
     <dc:date>2021-01-05</dc:date>
-    <dc:description>The spine XML is invalid. The reading system must not display the navigation document.</dc:description>
-    <dc:identifier id="pub-id">confreq-rs-xml-nval_invalid</dc:identifier>
+    <dc:description>This is a test of XML processing. The nav doc should not appear in the spine because the itemref is commented out.</dc:description>
+    <dc:identifier id="pub-id">confreq-rs-xml-nval_comment</dc:identifier>
     <dc:language>en</dc:language>
-    <dc:title>Invalid XML navigation document</dc:title>
+    <dc:title>XML Processing of Comments</dc:title>
```

Comment:
Sentence case for titles, please, because it looks better at https://w3c.github.io/epub-tests/#sec-test-tables (since these are really short descriptions more than titles).